datakyt/db/import/insert_data_in_project_table.py

```diff
@@ -0,0 +1,23 @@
+import csv
+
+
+def insert_csv_file_to_database(path, conn):
```

Comment:
Rename it to `insert_into_project_table`. Rename `py` file too.

Reply:
done